apps/dashboard/app/assets/stylesheets/common.scss

```diff
@@ -21,3 +20,8 @@
   z-index: 1100;
   background-color: inherit;
 }
+
+.5-rem {
```

Comment:
I tried the same lol. I don't think CSS classes can start with a number. Should be `rem-5` instead.